conf/nginx-nr-agent.ini

```diff
@@ -0,0 +1,46 @@
+# global settings
```

Comment:
more on the nginx-nr-agent and the dependencies can be found in the branch file https://github.com/muymoo/staticfile-buildpack/tree/files